Makefile

```diff
@@ -113,3 +105,31 @@ publish: images
 			docker push $(REGISTRY)/$${cmd}:$${tag}; \
 		done ; \
 	done
+
+
+# Code generation targets
+#########################
+CODEGEN_APIS_VERSIONS := mysql:v1alpha1
+CODEGEN_TOOLS := deepcopy client lister informer openapi
+CODEGEN_APIS_PKG := $(PACKAGE_NAME)/pkg/apis
```

Comment:
`$(PACKAGE_NAME)/pkg/apis` is the default for `CODEGEN_APIS_PKG`. No need to override.